app/scripts/metamask-controller.test.js

```diff
@@ -4204,7 +4204,7 @@ describe('MetaMaskController', () => {
       });
 
       it('returna config when not matching default mapping', async () => {
```

Comment:
Nit (though not introduced by this PR):
```suggestion
      it('returns config when not matching default mapping', async () => {
```